client/landing/gutenboarding/onboarding-block/create-site/index.tsx

```diff
@@ -28,7 +28,7 @@ const CreateSite: FunctionComponent< {} > = () => {
 	];
 
 	return (
-		<div className="create-site__background">
+		<div className="gutenboarding-page create-site__background">
```

Comment:
all the classes added to the wrapped components are not needed since they will be handled dynamically by `react-transition-group`